spec/rubocop/cop/lint/redundant_safe_navigation_spec.rb

```diff
@@ -14,6 +14,12 @@
     RUBY
   end
 
+  it 'does not register an offense and corrects when `&.` is used for snake case const receiver' do
+    expect_no_offenses(<<~RUBY)
+      FOO&.do_something
```

Comment:
Btw, I guess there's a small change that a class name will be one word, all caps - e.g. `URL`, `URI`, etc. I guess we can leave it as is, but are test cases for `CONST_NAME` and `Const_name`, so it's clearer what we are testing for exactly.

Reply:
Sure! I've updated the test cases.